README.md

```diff
@@ -68,6 +68,12 @@ Default: `100`
 
 Max count of items in a single chunk.
 
+### S3PIPELINE_PROCESS_ITEM_TIMEOUT (Optional)
```

Comment:
I think there could be a better alternative name of the setting. A name ending with `_TIMEOUT` sounds like a something related to error handling like [DNS_TIMEOUT](https://docs.scrapy.org/en/2.4/topics/settings.html#dns-timeout) or [DOWNLOAD_TIMEOUT](https://docs.scrapy.org/en/2.4/topics/settings.html#download-timeout).

However, I don't have any good idea of alternative name. It's OK just like that.

Reply:
May be `S3PIPELINE_MAX_WAIT_UPLOAD_TIME` is better